extensions-core/kubernetes-extensions/pom.xml

```diff
@@ -96,7 +96,7 @@
     <dependency>
       <groupId>org.bouncycastle</groupId>
       <artifactId>bcprov-ext-jdk15on</artifactId>
-      <version>1.68</version>
+      <version>1.69</version>
```

Comment:
why are we only updating `bcprov-ext-jdk15on` and not `bcprov-jdk15on` ?
We should also move to 1.70 since that is out already